audit/projects/k8s-cip-test-prod/services/logging/metrics.json

```diff
@@ -1 +0,0 @@
-[]
```

Comment:
Expected, this is a "filter out noise" fix from https://github.com/kubernetes/k8s.io/pull/2133